_pages/how-we-work/tools/slack.md

```diff
@@ -60,6 +59,28 @@ To invite people to a public Slack channel, send them to [chat.18f.gov](https://
 
 See [the instructions](https://github.com/18F/chat#adding-new-public-channels).
 
+## Offboarding
+
+Slack users need to be offboarded in the following situations:
+
+* Someone leaving TTS
+    * Someone from [#talent](https://gsa-tts.slack.com/messages/talent/) should post in [#admins-slack](https://gsa-tts.slack.com/messages/admins-slack/) when this happens.
+* A partner who's employment/engagement is ending
+    * It is the responsibility of the engagement manager to inform [#admins-slack](https://gsa-tts.slack.com/messages/admins-slack/) when this is happening.
+
+An admin will then do one of the following:
+
+* **[Convert them to a Single-Channel Guest.](https://get.slack.help/hc/en-us/articles/218124397-Change-a-team-member-s-role)**
+    * Put them in one of the following channels (as appropriate):
+        * [#alumni](https://gsa-tts.slack.com/messages/alumni/) (former Full Members only)
+        * One of the `-public` channels (their pick)
+    * [Disable their 2FA](https://get.slack.help/hc/en-us/articles/212221668-Require-two-factor-authentication-for-your-team).
+        * ...unless you know they were using their personal device for 2FA — we don't want them to be locked out.
+    * [Change their email](https://get.slack.help/hc/en-us/articles/225531168-Change-a-team-member-s-email-address) to their personal address
+* **[Deactivate their account.](https://get.slack.help/hc/en-us/articles/204475027-Deactivate-a-team-member-s-account)**
```

Comment:
Sounds good to me.  I check it out in Federalist and everything looks good.  Ready to merge?